src/sergittos/bedwars/session/Session.php

```diff
@@ -284,7 +284,9 @@ public function hideBossBar(): void {
     }
 
     public function sendDataPacket(ClientboundPacket $packet): void {
-        $this->player->getNetworkSession()->sendDataPacket($packet);
+        if(Server::getInstance()->getPlayerExact($this->player->getName()) !== null and Server::getInstance()->getPlayerExact($this->player->getName())->getNetworkSession() !== null){
```

Comment:
Please change this to
```suggestion
        if ($this->player->isConnected() && $this->player->spawned) {
```